src/App.tsx

```diff
@@ -15,36 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SORT {
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+  DEFAULT = '',
+}
+
+function sort(goods: string[], sortField: SORT, isReversed: boolean): string[] {
```

Comment:
According to the checklist, direct interaction with the DOM should be avoided. However, this function seems to be a pure function that doesn't interact with the DOM but rather sorts an array, which is a good practice in React.